ricecooker/utils/downloader.py

```diff
@@ -197,7 +207,7 @@ def read(
 
 
 def make_request(
-    url, clear_cookies=False, headers=None, timeout=60, session=None, *args, **kwargs
+    url, clear_cookies=False, headers=None, timeout=60, session=None, user_email=None, *args, **kwargs
```

Comment:
I don't think we need to pass this in, we instead copy the user agent from the globally configured DOWNLOAD_SESSION that should have had its user agent headers set properly. So far this PR is not doing this though, so is missing the main point of this issue.